charts/kong/README.md

```diff
@@ -772,6 +772,16 @@ containerSecurityContext: # run as root to bind to lower ports
   runAsUser: 0
 ```
 
+**Note:** The default pod annotations in this chart disables mesh proxing for Kuma and istio mesh.
+If you don't want to run Kong with disabled proxing inside this meshes, you can tweak the `podAnnotations` configurations as in the example:
+
+```yaml
+# Enable mesh proxing for kuma and istio
+podAnnotations:
+  kuma.io/gateway: disabled
+  traffic.sidecar.istio.io/includeInboundPorts: "*"
+```
```

Comment:
Can you clarify the intent here/environments you'd expect to use this configuration, and and share any test results you have for it?

The standard configuration disables only inbound mesh proxying (`gateway` is basically [Kuma's alias for exclude all inbound](https://kuma.io/docs/1.6.x/reference/kubernetes-annotations/#kuma-io-gateway)). Outbound proxying is still enabled, and as far as we know that makes sense for our expected topology where Kong serves as an edge gateway.

The explanation here suggests that the defaults disable the mesh proxies entirely. I expect that the intent here is to provide a configuration for topologies where Kong is somewhere in the center of the cluster network, but the explanation doesn't indicate as much.

Reply:
You're right. Here's a new attempt, with a somewhat more detailed explanation of these annotations.